### PR TITLE
NEW (Extension) @W-16095938@ Remove files and directories not required by vsix

### PR DESCRIPTION
**Pull request by jag-j**

### CODE REVIEW

File: .vscodeignore [hunk 1]
```diff
@@ -8,3 +8,8 @@ vsc-extension-quickstart.md
 **/.eslintrc.json
 **/*.map
 **/*.ts
+CODEOWNERS
+code-fixtires/**
```
Comment (jfeingold35) [inline .vscodeignore:12]: Typo, should be `code-fixtures`.

File: .vscodeignore [hunk 2]
```diff
@@ -8,3 +8,8 @@ vsc-extension-quickstart.md
...
+coverage/**
+github-actions/**
```
Comment (jfeingold35) [inline .vscodeignore:14]: If we're putting `github-actions` here, do we also want to put `.git2gus` and `.github`?

**Review by jfeingold35 — CHANGES_REQUESTED** [review]

File: .vscodeignore (hunk 1 above)
Comment (jag-j) [reply .vscodeignore]: Fixed

File: .vscodeignore (hunk 2 above)
Comment (jag-j) [reply .vscodeignore:14]: Yes, they can go away as well.

**Review by jfeingold35 — APPROVED** [review]

**Pull request merged**